content/mozilla/2015.055.contineous.token.rst

```diff
@@ -0,0 +1,470 @@
+Comment passer de LIMIT / OFFSET aux Continuation Tokens ?
+##########################################################
+
+:date: 2015-04-16 13:25
+:authors: Rémy Hubscher
+:summary: De LIMIT / OFFSET aux continuations tokens
+
+
+Si vous êtes là, c'est que quelqu'un a du vous le dire : Paginer sur des
+grosses tables avec LIMIT et OFFSET ça ne monte pas en charge et plus
+on avance dans les pages, plus c'est lent.
+
+Il y a même une initiative "STOP OFFSET" pour que les développeurs y
+pensent : http://use-the-index-luke.com/no-offset
+
+
+Pourquoi c'est mal d'utiliser OFFSET et LIMIT ?
+===============================================
+
+Le post ci-dessus l'explique mieux que moi, mais ce qu'il faut retenir
+c'est que la base de données doit faire la requête entière pour
+ensuite aller se déplacer jusqu'à la bonne ligne.
+
+Si pendant que vous changez de page quelqu'un modifie la collection
+vous pouvez aussi louper des entrées ou avoir des doublons dans les
```

Comment:
Ça on sait pas le résoudre autrement qu'en forçant le tri sur un timestamp unique, non ?